src/lib/hooks/useNetworkFees.ts

```diff
@@ -0,0 +1,198 @@
+import { Coins, Services } from '@ardenthq/sdk';
+import { Contracts } from '@ardenthq/sdk-profiles';
+import { useCallback, useEffect, useState } from 'react';
+import { BigNumber } from '@ardenthq/sdk-helpers';
+import { useEnvironmentContext } from '@/lib/context/Environment';
+
+export interface TransactionFees {
+    static: string;
+    max: string;
+    min: string;
+    avg: string;
+    isDynamic?: boolean;
+}
+
+interface CreateStubTransactionProperties {
+    coin: Coins.Coin;
+    getData: (wallet: Contracts.IReadWriteWallet) => Record<string, any>;
+    stub: boolean;
+    type: string;
+}
+
+interface CalculateBySizeProperties {
+    coin: Coins.Coin;
+    data: Record<string, any>;
+    type: string;
+}
+
+interface CalculateProperties {
+    coin: string;
+    data?: Record<string, any>;
+    network: string;
+    type: string;
+}
+
+export const useNetworkFees = ({
+    profile,
+    coin,
+    network,
+    type,
+    data,
+}: {
+    profile: Contracts.IProfile;
+    coin: string;
+    network: string;
+    type: string;
+    data?: Record<string, any>;
+}) => {
+    const [isLoadingFee, setIsLoadingFee] = useState<boolean>(false);
+    const [fees, setFees] = useState<TransactionFees>();
+    const { env } = useEnvironmentContext();
+
+    const getMuSigData = (senderWallet: Contracts.IReadWriteWallet, data: Record<string, any>) => {
```

Comment:
we don't need musig related stuff at this time, it's not supported in arkconnect